docs/developer-docs/web-apps/verifiable-credentials/overview.mdx

```diff
@@ -0,0 +1,47 @@
+---
+keywords: [intermediate, concept, identity, credentials, verifiable credentials]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Overview
+
+<MarkdownChipRow labels={["Intermediate", "Concept"]} />
+
+A credential is a document or certificate that validates an individual's identity or qualifications. Credentials are a part of daily life, with common examples including:
+
+- A driver's license that qualifies an individual to operate a motor vehicle.
+- A university degree that can be used to assert a level of education.
+- A government-issued passport enabling travel between countries.
+
+These are examples of physical credentials, as they have a physical component that must be presented for validation. Credentials can also be digital, relying on authentication using certifications or digital signatures for validation, such as a username and password or ICP Internet Identity.
+
+Verifiable credentials (VCs) are a type of digital credential that enables individuals, organizations, or things (such as devices) to cryptographically prove the claims they make about themselves.
+
+The technical implementation of VCs on ICP can be found in the [specification](https://github.com/dfinity/internet-identity/blob/main/docs/vc-spec.md). It is recommended you first read the rest of this documentation, as it includes technical details.
+
+## Benefits
+
+Verifiable credentials are designed to be tamper-evident, privacy-preserving, and interoperable across different systems and contexts. By using verifiable credentials, you will have a mechanism to express these sorts of credentials on ICP in a way that is cryptographically secure, privacy-preserving, and machine-verifiable.
+
+## Terminology
+
+When learning about and using VCs, there are two primary terms to familiarize yourself with:
+
+- **Issuer**: A service or app that issues verifiable credentials to a user.
+
+- **Relying party**: A service or app that requests users provide verifiable credentials.
+
+## How to use verifiable credentials
+
+To use verifiable credentials there are two workflows. First, you can [become an issuer](issuer.mdx) of verifiable credentials. By issuing verifiable credentials, you can empower users to control and share their credentials selectively, enhancing privacy and giving users greater autonomy over their digital identity.
+
+The second workflow allows you to [become a relying party](relying-party.mdx) for verifiable credentials. By requesting verifiable credentials from your application's users or customers, you can ensure that the claims that users make are verified by parties (issuers) that you trust, while preserving the user’s privacy.
```

Comment:
I don't understand the second sentence beginning: "By requesting verifiable credentials from your application's users or customers"

I believe that the message we give throught the document is more "an app or user request credentials" not "request credentials from users".

If so, we could change it to:
> By requesting verifiable credentials to issuers, you can ensure that the claims ...

Reply:
Revised to "By requesting verifiable credentials from issuers.."

Reply:
I prefer what's written to @lmuntaner's suggestion. The user must consent to sharing the credential with the relying party, therefore the relying party is requesting the credential from the user, even though the issuer is providing the credential.

Reply:
I see your point @marydwyer . How about we change the "requesting",  believe that's the part that we use technically and might confuse further reading. How about:

> By requiring verifiable credentials from your application's users ...

Reply:
Revised to "By requiring verifiable credentials from your application's users.."